newsfragments/48027.significant.rst

```diff
@@ -0,0 +1,14 @@
+``password`` extra has been removed
```

Comment:
Removal of a dependency is a breaking change?

Reply:
I'd say this is a gray area. Figured it doesn't hurt to at least track it so we can decide as we are writing the release notes.

Reply:
Sounds good. Then we should select "Dependency changes"

Reply:
Idk, it's more of a dependency removal, it's doesn't break anything. You can still ask for the extra too, and pip/uv will just warn 🤷.